### PR TITLE
feat #871: Create new fe app

### LINKED ISSUE #871: Spaces app setup

### PR DESCRIPTION
**Pull request by Kasia9090**
### Set up for spaces app including:
 - sentry
 - apollo client
 - graphQL codegen
 - ui-kit
 
 ### what does not work / is missing:
 
 - ory auth does not work
 - sentry source map upload has to be set up by vercel (blocked by this mr)
 - new project needs to be added on vercel (blocked by this mr)
 

What types of changes does your code introduce?  _Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Build related changes
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Other (please describe below)

### CODE REVIEW

File: packages/apps/spaces/pages/api/.ory/[...paths].ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// Copyright © 2022 Ory Corp
```
Comment (edifirut) [inline packages/apps/spaces/pages/api/.ory/[...paths].ts:1]: this file is duplicated

File: packages/apps/spaces/src/graphQL/hooks.ts [hunk 2]
```diff
@@ -0,0 +1,86 @@
+import { gql } from '@apollo/client';
```
Comment (edifirut) [inline packages/apps/spaces/src/graphQL/hooks.ts:1]: Please add a README.md to the project for the generation of the graphQL files

**Review by edifirut — APPROVED** [review]

File: packages/apps/spaces/pages/api/.ory/[...paths].ts (hunk 1 above)
Comment (Kasia9090) [reply packages/apps/spaces/pages/api/.ory/[...paths].ts:1]: https://github.com/openline-ai/openline-customer-os/issues/874

File: packages/apps/spaces/src/graphQL/hooks.ts (hunk 2 above)
Comment (Kasia9090) [reply packages/apps/spaces/src/graphQL/hooks.ts:1]: https://github.com/openline-ai/openline-customer-os/issues/875

**Pull request merged**